Formula/nest.rb

```diff
@@ -0,0 +1,158 @@
+class Nest < Formula
+  desc "The Neural Simulation Tool (NEST) with Python2 bindings (PyNEST)"
+  homepage "http://www.nest-simulator.org/"
+  url "https://github.com/nest/nest-simulator/archive/v2.14.0.tar.gz"
+  sha256 "afaf7d53c2d5305fac1257759cc0ea6d62c3cebf7d5cc4a07d4739af4dbb9caf"
+
+  bottle do
+    sha256 "4f7d765cd75ae3272ce15bb293c20a6cdfc9ccb551f0a07d00016c75ae0ef267" => :high_sierra
+    sha256 "60e4b0595c5d11003ffe1eca3fdca3c197b8711336ead21eba67620db96454c0" => :sierra
+    sha256 "8a4c8e6d0d2e4ce93d76af73197c693f8c303095800e82b7266b922848f1a39b" => :el_capitan
+    sha256 "6207dea13f90b17a95313b964323f7971cd5e2507a38b0ffd5d64b2f346e7851" => :x86_64_linux
+  end
+
+  # Add patches for matplotlib removal in upstream:
+  patch do
+    url "https://github.com/nest/nest-simulator/commit/b7f3313c0558da0da269cd7a11cc657da2049df7.patch?full_index=1"
+    sha256 "54588322953d0dbc02a036053cdca7ad47153eb46464ce471579adb0f958277d"
+  end
+
+  patch do
+    url "https://github.com/nest/nest-simulator/commit/d60e189123b994764bef8179aa91b4b001dad1dd.patch?full_index=1"
+    sha256 "44c14588c1b9b8fcc67326c4119e8d42c8a069670630eab5a1522daac2219708"
+  end
+
+  patch do
+    url "https://github.com/nest/nest-simulator/commit/5cfa2a77a060eae0f90182c56c42bdc8549dd19c.patch?full_index=1"
+    sha256 "69e4d642555a7abefc65dc287d48715ffb42faac72e1ff3dfb880fda58564045"
+  end
+
+  option "with-python", "Build Python3 bindings (PyNEST) instead of Python2 bindings."
```

Comment:
Let's choose a python version, and build it with no options. I recommend moving to python3.